src/rdkafka_cgrp.c

```diff
@@ -5274,6 +5312,9 @@ rd_kafka_cgrp_modify_subscription(rd_kafka_cgrp_t *rkcg,
 static rd_kafka_resp_err_t rd_kafka_cgrp_unsubscribe(rd_kafka_cgrp_t *rkcg,
                                                      rd_bool_t leave_group) {
 
+        /**
+         * TODO: Update debug with Regex changes for KIP-848
+         */
```

Comment:
If we do the change below to avoid modfying `rkcg_subscription` we don't need to change the logging part.